src_c/key.c

```diff
@@ -106,8 +150,12 @@ static PyTypeObject pgScancodeWrapper_Type = {
     PyVarObject_HEAD_INIT(NULL, 0).tp_name = _PG_SCANCODEWRAPPER_TYPE_FULLNAME,
     .tp_repr = (reprfunc)pg_scancodewrapper_repr,
     .tp_as_mapping = &pg_scancodewrapper_mapping,
-    .tp_flags =
-        Py_TPFLAGS_DEFAULT | Py_TPFLAGS_TUPLE_SUBCLASS | Py_TPFLAGS_BASETYPE,
+    .tp_iter = (getiterfunc)pg_iter_raise,
+    .tp_iternext = (iternextfunc)pg_iter_raise,
+#ifdef PYPY_VERSION
+    .tp_new = pg_scancodewrapper_new,
+#endif
+    .tp_flags = Py_TPFLAGS_DEFAULT | Py_TPFLAGS_BASETYPE,
```

Comment:
hmmm, removing the `Py_TPFLAGS_TUPLE_SUBCLASS` flag from here is probably not a good idea. This wrapper could be expected by user code to be a subclass tuple.

An alternate solution that I can think of, is making `tp_new` also error (because I don't think users would want to instantiate a scancodewrapper themselves) and instead create an internal new function at the C level that `key_get_pressed` can use

Reply:
Oops, that was accidental, good eye